pywinauto/win32_hooks.py

```diff
@@ -542,7 +547,7 @@ def _mouse_ll_cb(code, event_code, mouse_data_ptr):
             self.mouse_id = windll.user32.SetWindowsHookExA(
                 win32con.WH_MOUSE_LL,
                 _mouse_ll_cb,
-                windll.kernel32.GetModuleHandleA(None),
+                win32api.GetModuleHandle(None),
```

Comment:
`windll.kernel32.GetModuleHandleA.argtypes = [wintypes.LPCSTR]`

`LPCSTR` is alias for `char *`.

`GetModuleHandle` is a macro which will expand to either `GetModuleHandleA` or `GetModuleHandleW`. I think is is better to explicitly use `GetModuleHandleA` function for `SetWindowsHookExA` and `GetModuleHandleW` function for `SetWindowsHookExW`.

That way the types should always be correct.

Reply:
The difference is that `win32api` version is determined at compile time in pyWin32. So there is no `A` or `W` version, only alias which maps to ANSI or Unicode one at pyWin32 build. Using `None` is more correct and explicit for pyWin32 version of this function. So we shouldn't care about `LPCSTR` or `LPCWSTR`.